rest/pom.xml

```diff
@@ -18,6 +18,16 @@
             <groupId>${project.groupId}</groupId>
             <artifactId>spi</artifactId>
         </dependency>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>rest</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.n52.series-api.db</groupId>
+            <artifactId>dao</artifactId>
```

Comment:
don' mix SPI implementation with SPI, i.e. in every case the SPI is independent from its implementations (otherwise all implementations depend on each other)